nucliadb_models/nucliadb_models/search.py

```diff
@@ -886,22 +947,27 @@ class ChatRequest(BaseModel):
     )
     autofilter: bool = SearchParamDefaults.autofilter.to_pydantic_field()
     highlight: bool = SearchParamDefaults.highlight.to_pydantic_field()
-    resource_filters: List[
-        str
-    ] = SearchParamDefaults.resource_filters.to_pydantic_field()
+    resource_filters: List[str] = (
+        SearchParamDefaults.resource_filters.to_pydantic_field()
+    )
     prompt: Optional[str] = SearchParamDefaults.prompt.to_pydantic_field()
     citations: bool = Field(
         default=False,
         description="Whether to include the citations for the answer in the response",
     )
-    security: Optional[
-        RequestSecurity
-    ] = SearchParamDefaults.security.to_pydantic_field()
+    security: Optional[RequestSecurity] = (
+        SearchParamDefaults.security.to_pydantic_field()
+    )
     rag_strategies: list[RagStrategies] = Field(
         default=[],
         title="RAG context building strategies",
         description="Options for tweaking how the context for the LLM model is crafted. `full_resource` will add the full text of the matching resources to the context. `field_extension` will add the text of the matching resource's specified fields to the context. If empty, the default strategy is used.",  # noqa
     )
+    rag_images_strategies: list[RagImagesStrategies] = Field(
+        default=[],
+        title="RAG image context building strategies",
+        description="Options for tweaking how the image based context for the LLM model is crafted. `page_image` will add the full page image of the matching resources to the context. If empty, the default strategy is used with the image of the paragraph.",  # noqa
```

Comment:
You should mention the `paragraph_image` strategy too

Reply:
Its also images of the page